README.md

```diff
@@ -155,6 +155,9 @@ A quick start for specific scenarios is below:
 * Set `SendKey` to a valid Honeycomb Key
 * Set `SendKeyMode` to `listedonly`
 
++**NOTE**
++
++When using Beelines with a classic API key to send data to Refinery, ensure that the `SendKey` is also a classic Key, not an environment and service(E&S) key.
```

Comment:
Would that work tho or would that header still get dropped? I think if you're sending with a Classic Key/Header combo that you should not use SendKey at all.

Reply:
If the `SendKey` is a classic key, it should work since the `dataset` header is present